doc/classes/TextEdit.xml

```diff
@@ -83,15 +83,15 @@
 				Adds a selection and a caret for the next occurrence of the current selection. If there is no active selection, selects word under caret.
 			</description>
 		</method>
-		<method name="adjust_carets_after_edit">
+		<method name="adjust_carets_after_edit" is_deprecated="true">
 			<return type="void" />
 			<param index="0" name="caret" type="int" />
 			<param index="1" name="from_line" type="int" />
 			<param index="2" name="from_col" type="int" />
 			<param index="3" name="to_line" type="int" />
 			<param index="4" name="to_col" type="int" />
 			<description>
-				Reposition the carets affected by the edit. This assumes edits are applied in edit order, see [method get_caret_index_edit_order].
+				[i]Obsoleted.[/i] No longer necessary since methods now adjust carets themselves.
```

Comment:
```suggestion
				[i]Deprecated.[/i] No longer necessary since methods now adjust carets themselves.
```
Agreed upon syntax in new content, and we use the same wording regardless if it's now non-functional or just slated for future removal